plugin/PlayAssetDelivery/src/androidTest/java/com/google/play/godotassetdelivery/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.google.play.godotassetdelivery;
```

Comment:
The namespace of the asset delivery code inside of play core is com.google.play.core.assetpacks.
Might be good to change this package name to match. How about `com.google.play.core.godot.assetpacks`? That way if we add other play core features they can all live under the `com.google.play.core.godot` package.

Reply:
Yeah that make sense. This also adds readability given that the current godotassetdelivery name is too long.